src/@next/Icon/icons/ri-calendar-event-line.svg

```diff
@@ -0,0 +1,6 @@
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
can you update to `viewBox="0 0 24 24"` to follow the standard

Reply:
please do the same with the others as well

Reply:
Updated

Reply:
Is this okay? It looks like the icon size is not covering the container
https://www.figma.com/file/xaLr9U0h5JyIJ9p0sjBXiF/Employers---Dashboard?node-id=1501%3A169285&t=dZtijGpFl7mXySFy-1

Reply:
that's weird. i tried exporting one of them, it got 24 by default though 🤔 did you try reexporting from the "group" layer?

Reply:
Rectified